sweep/sweeper.go

```diff
@@ -102,7 +102,19 @@ func (s *Sweeper) GetSweepFee(ctx context.Context,
 
 	// Calculate weight for this tx.
 	var weightEstimate input.TxWeightEstimator
-	weightEstimate.AddP2WKHOutput()
+	switch destAddr.(type) {
+	case *btcutil.AddressWitnessScriptHash:
+		weightEstimate.AddP2WSHOutput()
+	case *btcutil.AddressWitnessPubKeyHash:
+		weightEstimate.AddP2WKHOutput()
+	case *btcutil.AddressScriptHash:
+		weightEstimate.AddP2SHOutput()
+	case *btcutil.AddressPubKeyHash:
```

Comment:
Nit: I don't think we support these addresses so can be removed?

Reply:
This address can be specified externally, on the `loop` command line.